### ISSUE TITLE
Using `set-page-properties` adds paragraph break

### ISSUE THREAD

**Issue by Alex-Muirhead** [ntjess/typst-drafting#12]
When calling `set-page-properties`, an extra paragraph break is added after the function call. 
This becomes a problem frequently when rotating pages using `page(flipped: true)`, as the page properties for drafting need to be updated to work properly. However the extra paragraph break added will mess with my formating (reducing available vertical space, etc). Is there any way to fix this or avoid this problem in the meantime? 
Thank you!

Simple example without calling
```typst
#import "@preview/drafting:0.2.0" : *
#set page(paper: "a6")

// #set-page-properties()

= An example heading

#lorem(100)
```
![Bug Testing](https://github.com/ntjess/typst-drafting/assets/15205931/25dabe6f-d175-438e-a34d-441e073bacaa)

Simple example with call
```typst
#import "@preview/drafting:0.2.0" : *
#set page(paper: "a6")

#set-page-properties()

= An example heading

#lorem(100)
```
![Bug Testing(1)](https://github.com/ntjess/typst-drafting/assets/15205931/782e5570-4c23-4c67-a3e3-3ca9ee4952b6)

**Comment by ntjess** [comment]
Thanks for reporting -- this is a known issue but I haven't had time to create a release. In the meantime, you can work around it with `place`:
```typst
#place(set-page-properties())
```

**Comment by ntjess** [comment]
Fixed in 769fbbc2909ad28f346ef8ce4d5c75afd11d4739
